### PR TITLE
Adding task7, 8, & 9

### PR DESCRIPTION
**Pull request by MartaSzymula**
Solving issues: #2 #3 #4

### CODE REVIEW

File: script_01.py [hunk 1]
```diff
@@ -79,12 +79,89 @@ def task6(a=[1,2,12,4], b=[2,4,2,9]):
...
+    '3' : "three",
+    '4' : "four",
+    '5' : "five",
+    '6' : "six",
+    '7' : "seven",
+    '8' : "eight",
+    '9' : "nine"
+    }
+
+    a_length = str(to_replace)
```
Comment (WojciechKusa) [inline script_01.py]: `a_length` is not the best name

File: script_01.py [hunk 2]
```diff
@@ -79,12 +79,89 @@ def task6(a=[1,2,12,4], b=[2,4,2,9]):
...
+    for n in range (0, len(list_elements)):
+        if list_elements[n] in numbers.values():
+            text = text + ' ' + str(list_elements[n])
+
+        else:
+            for i in range (0, len(list_elements[n])):
+                if list_elements[n][i] in numbers:
+                    text = text + ' ' + str(numbers[list_elements[n][i]])
+
+    text = text.strip()
```
Comment (WojciechKusa) [inline script_01.py:120]: why do you use `strip()`?

File: script_01.py [hunk 3]
```diff
@@ -79,12 +79,89 @@ def task6(a=[1,2,12,4], b=[2,4,2,9]):
...
+        Returned object is average of given parameters.
+    """
+    num_list = []
+    if type(x1) is list:
+        num_list = x1
+    else:
+        num_list = [x1, x2]
+    list_sum = 0
+    for i in range(0, len(num_list)):
+        num_float=float(num_list[i])
```
Comment (WojciechKusa) [inline script_01.py]: please add space around `=`

**Review by WojciechKusa — COMMENTED** [review]
good, please fix minor changes

File: script_01.py (hunk 2 above)
Comment (MartaSzymula) [reply script_01.py:120]: It removes the space in the begging of the text

Comment (WojciechKusa) [reply script_01.py:120]: +1

**Pull request merged**